app/src/main/java/com/poupa/vinylmusicplayer/dialogs/BottomSheetDialog/EnqueueSongsBottomSheetDialog.java

```diff
@@ -0,0 +1,36 @@
+package com.poupa.vinylmusicplayer.dialogs.BottomSheetDialog;
+
+
+import java.util.List;
+
+import com.poupa.vinylmusicplayer.util.PreferenceUtil;
+
+
+public class EnqueueSongsBottomSheetDialog extends ButtonListBottomSheetDialog {
+    public static EnqueueSongsBottomSheetDialog newInstance() { return new EnqueueSongsBottomSheetDialog(); }
+
+    int defaultChoice;
+
+    public EnqueueSongsBottomSheetDialog setDefaultChoice(int defaultIndex) {
+        this.defaultChoice = defaultIndex;
+        return this;
+    }
+
+    @Override
+    public EnqueueSongsBottomSheetDialog setTitle(String title) {
```

Comment:
I dont get the reason of this override: you just call super's implementation, in this case there is no need for override, exact?

Reply:
I need this or setTitle will return a BottomSheetDialogWithButtons object and I will not be able to call setDefaultChoice after calling setTitle
Not sure if the override is necessary, seems cleaner at least

Reply:
This still bothers me, but I can live with it for now.
Probably when we want to apply this bottom sheet to other dialogs, we need to revisit this to avoid (possible) proliferation of trivial overriden methods.